README.md

```diff
@@ -169,18 +169,23 @@ To run Bats' internal test suite (which is in the container image at
 
     $ docker run -it bats/bats:latest /opt/bats/test
 
-To run a test suite from your local machine, mount in a volume and direct Bats
-to its path inside the container:
+To run a test suite from a directory called `test` in the current directory of
+your local machine, mount in a volume and direct Bats to its path inside the
+container:
 
-    $ docker run -it -v "$(pwd):/opt/bats" bats/bats:latest /opt/bats/test
```

Comment:
I believe the intent of this line was actually to help users run Bats' own tests, not to run their own Bats tests (note the "To run Bats' internal test suite" line above).

It's certainly helpful to provide an example for running your own Bats tests, but this section still reads strangely.

Reply:
The code still works to test `bats-core`, but I take your point on the phrasing changes. 

Perhaps the sentence could be prefixed with: 

> To run arbitrary tests (this example still works for the `bats-core` directory, but mounted into the container instead of reading the files from the container image)

Wordier, but perhaps clearer?

Reply:
I'd suggest splitting the two purposes apart more fully; a section on "how to run your own tests" and a section on "how to test bats itself".